test/test_google_cloud_iam.py

```diff
@@ -0,0 +1,128 @@
+
+from cirrus.google_cloud.iam import (
+    GooglePolicyMember,
+    GooglePolicyRole,
+    GooglePolicyBinding,
+    GooglePolicy,
+)
+
+def test_create_google_policy():
+
+    role1 = GooglePolicyRole("roles/admin")
+    mem11 = GooglePolicyMember("user", "mem11@gmail.com")
+    mem12 = GooglePolicyMember(
+        "serviceAccount", "mem12@iam.gserviceaccount.com")
+    mem13 = GooglePolicyMember("user", "mem13@gmail.com")
+
+    role2 =  GooglePolicyRole("roles/read")
+    mem21 = GooglePolicyMember("user", "mem21@gmail.com")
+    mem22 = GooglePolicyMember(
+        "serviceAccount", "mem22@iam.gserviceaccount.com")
+    mem23 = GooglePolicyMember("user","mem23@gmail.com")
+
+    mems1 = [mem11, mem12, mem13, mem23]
```

Comment:
why is `mem23` in `mems1`? seems like by naming convention `mems1` group should only have `mem1x` members

Reply:
I wanted to have one member be in both policy bindings. But the naming is confusing here, so i'll clean that up.